aten/src/ATen/native/Embedding.cpp

```diff
@@ -24,12 +24,26 @@ Tensor embedding(const Tensor & weight, const Tensor & indices,
     return weight.index_select(0, indices);
   }
 
-  auto size = indices.sizes().vec();
-  for (auto d : weight.sizes().slice(1)) {
+  auto size = indices.sym_sizes().vec();
```

Comment:
@ezyang are these changes not needed at all?  I realized later that backward was really what was tripping us up,  and fixed that.  I confirmed that even if I revert these changes bert is happy (or, still reports the 'next error').

Reply:
i guess, bc we have a decomp registered for embedding.  

does that mean in general we should just avoid messing with the c++ side? i can drop this part.

Reply:
yes you don't need this, the python decomp will be preferred so we will never exercise this.